### PR TITLE
remove: fluent

### PR DESCRIPTION
**Pull request by afonsojramos**
As per https://github.com/williamckha/spicetify-fluent/issues/55, it doesn't make sense to keep Fluent in this repo if it is not being actively developed here but instead on the repo referred before.

### CODE REVIEW

**Comment by afonsojramos** [comment]
That's why I added it to the `spicetify-docs` repo though @theRealPadster

**Comment by williamckha** [comment]
@afonsojramos @theRealPadster I think it's fine if we remove Fluent from this repo. Most users of this repo probably clone the whole repo expecting to have the files from each theme ready to install, so just having a link in the readme of the theme doesnt make sense for the purpose of this collection

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ You can add your own theme simply by opening a Pull Request (more info in [CONTR
 ## Notes:
 
 *   **These themes require you to have the latest version of Spotify and Spicetify.**
-*   **To install Dribbblish, Turntable and Fluent themes - follow the instructions in their READMEs**.
+*   **To install Dribbblish, Turntable themes - follow the instructions in their READMEs**.
```
Comment (theRealPadster) [inline README.md:12]:
```suggestion
*   **To install Dribbblish and Turntable themes, follow the instructions in their READMEs**.
```

**Comment by CharlieS1103** [comment]
@afonsojramos Make sure you remove it from the manifest.json as well.

**Review by theRealPadster — APPROVED** [review]

**Pull request merged**